pyvista/core/filters/poly_data.py

```diff
@@ -1563,22 +1564,30 @@ def decimate(
         self,
         target_reduction,
         volume_preservation: bool = False,
-        attribute_error: bool = False,
-        scalars: bool = True,
-        vectors: bool = True,
-        normals: bool = False,
-        tcoords: bool = True,
-        tensors: bool = True,
+        # scalars: bool,  # kwargs for default value
+        # vectors: bool,  # kwargs for default value
+        # normals: bool,  # kwargs for default value
+        # tcoords: bool,  # kwargs for default value
+        # tensors: bool,  # kwargs for default value
```

Comment:
Use `None` and set value dynamically based on `enable_all_attribute_error`
```suggestion
        scalars: bool | None = None,
        vectors: bool | None = None,
        normals: bool | None = None,
        tcoords: bool | None = None,
        tensors: bool | None = None,
```